### DISCUSSION TITLE
Embed a html file inside my page

### DISCUSSION THREAD

**Discussion by imflash217** [squidfunk/mkdocs-material#3573] [Q&A]
Hi,
I have `.html` file exported using MARP (https://github.com/marp-team/marp/) slides.
I want to include that inside my mkdocs-material page.
How can I do this?
Thanks

**Comment by squidfunk** [comment]
This should be possible by using the [Snippets extension](https://facelessuser.github.io/pymdown-extensions/extensions/snippets/):

``` md
--8<-- "file.html"
```
